akka-remote/src/main/scala/akka/remote/artery/compress/CompressionProtocol.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.remote.artery.compress
+
+import akka.actor.{ ActorRef, Address }
+import akka.remote.artery.ControlMessage
+
+// FIXME serialization 
+/** INTERNAL API */
+object CompressionProtocol {
+
+  /** INTERNAL API */
+  sealed trait CompressionMessage
+
+  /**
+   * INTERNAL API
+   * Sent by the "receiving" node after allocating a compression id to a given [[akka.actor.ActorRef]]
+   */
+  private[remote] final case class ActorRefCompressionAdvertisement(from: Address, ref: ActorRef, id: Int)
```

Comment:
`from` should be `UniqueAddress`
ah, so currently each actor ref is advertised separately, I thought it was the whole table that should be advertised, but that is perhaps in later version when we add versioning of the tables